docs/docs/admin/installation.mdx

```diff
@@ -186,3 +186,92 @@ Then point your Ingress to the Anubis port:
               # diff-add
               name: anubis
 ```
+
+## Nginx
+
+The below is a very minimal Nginx config example to demonstrate how to
+insert Anubis in the middle of a standard nginx configuration.
+Do not forget to add `access_log`, `server_tokens`, etc. as usual.
+Also one does have to correct the paths, eg for SSL keys/certs.
+
+First we define `conf-anubis.inc` where we include standard Anubis configuration:
+```
+# Serve Anubis images directly instead of having it pass through to the service
+# Place `happy.webp`, `pensive.webp` and `reject.webp` in the directory.
+location /.within.website/x/cmd/anubis/static/img/
+{
+        alias /www/anubis/www/img/;
+}
+
```

Comment:
Maybe this snippet needs to go on another page in the admin docs. Eventually I want to make the images directly configurable in Anubis itself, but the branding being difficult to change is a feature right now as a way to incentivize the project being financially sustainable.